src/Tasks/Microsoft.NET.Build.Tasks/GenerateDepsFile.cs

```diff
@@ -112,16 +115,20 @@ protected override void ExecuteCore()
                     AssemblyName,
                     AssemblyExtension,
                     AssemblyVersion,
-                    AssemblySatelliteAssemblies);            
+                    AssemblySatelliteAssemblies);
 
             IEnumerable<ReferenceInfo> referenceAssemblyInfos =
                 ReferenceInfo.CreateReferenceInfos(ReferenceAssemblies);
 
             IEnumerable<ReferenceInfo> directReferences =
                 ReferenceInfo.CreateDirectReferenceInfos(ReferencePaths, ReferenceSatellitePaths);
 
+            IEnumerable<ReferenceInfo> dependencyReferences =
+                ReferenceInfo.CreateDependencyReferenceInfos(ReferenceDependencyPaths, ReferenceSatellitePaths);
```

Comment:
Does the satellite information for these `ReferenceDependencyPaths` come through `ReferenceSatellitePaths`?  It would be good to test this.

Reply:
Yes. The `ReferenceDependencyPaths` satellites come through `ReferenceSatellitePaths`.

Under the current code, the satellites are being adding to the project reference resources and the actual reference resources, so I'll need to fix up the logic slightly:
```json
"ClassLibrary1/1.0.0": {
  "dependencies": {
    "System.Runtime.CompilerServices.Unsafe": "4.4.0",
    "ClassLibrary2": "1.0.0.0",
    "ClassLibrary3": "1.0.0.0"
  },
  "runtime": {
    "ClassLibrary1.dll": {}
  },
  "resources": {
    "de-DE/ClassLibrary2.resources.dll": {
      "locale": "de-DE"
    },
    "de-DE/ClassLibrary3.resources.dll": {
      "locale": "de-DE"
    }
  }
},
"ClassLibrary2.Reference/1.0.0.0": {
  "runtime": {
    "ClassLibrary2.dll": {}
  },
  "resources": {
    "de-DE/ClassLibrary2.resources.dll": {
      "locale": "de-DE"
    }
  }
},
"ClassLibrary3.Reference/1.0.0.0": {
  "runtime": {
    "ClassLibrary3.dll": {}
  },
  "resources": {
    "de-DE/ClassLibrary3.resources.dll": {
      "locale": "de-DE"
    }
  }
}
```